sources/web/datalab/polymer/components/datalab-sessions/datalab-sessions.ts

```diff
@@ -217,7 +217,7 @@ class SessionsElement extends Polymer.Element implements DatalabPageElement {
     // notebook, then switches back to the sessions page. Killing a session also
     // triggers a refresh.
     if (!this._sessionListRefreshIntervalHandle) {
-      this._sessionListRefreshIntervalHandle = setInterval(() => {
+      this._sessionListRefreshIntervalHandle = window.setInterval(() => {
```

Comment:
Is this related to using locally installed build tools? If not, I would prefer to see it in a separate PR.